udaykalra/portfolio/src/main/webapp/script.js

```diff
@@ -28,6 +28,7 @@ function getCommentData() {
         const commentsElement = document.getElementById('quote-container');
         commentsElement.innerHTML = '';
         for (var inc = 0; inc < myObject.length; inc += 1) {
+          console.log(myObject);
```

Comment:
What does `myObject` contain? I'm wondering if it's name can be made more specific, so the reader would understand without reading through the rest of this method what will be logged.

Reply:
Done